arctic/store/_pandas_ndarray_store.py

```diff
@@ -253,8 +262,8 @@ def from_records(self, recarr):
         name = recarr.dtype.names[-1]
         return Series.from_array(recarr[name], index=index, name=name)
 
-    def can_write(self, version, symbol, data):
-        if isinstance(data, Series):
+    def can_write(self, version, symbol, data, **kwargs):
+        if isinstance(data, Series) and 'chunk_size' not in version:
```

Comment:
Why is `chunk_size` here? It doesn't seem right that PandasSeriesStore should know about the implementation of the chunked store.